code/modules/assembly/flash.dm

```diff
@@ -301,6 +301,17 @@
 	inhand_icon_state = "nullrod"
 	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	var/implant_message = "You don't remember anything before the start of this shift"
```

Comment:
```suggestion
	var/implant_message = "You don't remember anything before the start of this shift."
```

Reply:
imo this should probably just make them forget, like, the past 5 minutes

forgetting what happened BEFORE the shift isn't really useful for, like, anything other than maybe some RP elements. and seems like kind of the opposite of the men in black mindwiper?

Reply:
You're right, I kraft it to say'between'  not'before'and had a brain fart

Reply:
> imo this message should probably be changed to just make them forget, like, the past 5 minutes
> 
> forgetting what happened BEFORE the shift isn't really useful for, like, anything other than maybe some RP elements. and seems like kind of the opposite of the men in black mindwiper?

Yeah I meant it to say between, not before! My bad!